packages/react-components/react-menu/stories/Menu/MenuDescription.md

```diff
@@ -1,2 +1,4 @@
 A menu displays a list of actions. The Menu component handles the
 state management of the passed in list of actions.
+
+See also <a href="#" data-sb-kind="components-button-menubutton--default">MenuButton</a>
```

Comment:
This approach still breaks copying link URLs to share or open somewhere else, and also opening links in a new tab. That's not much worse than what we currently have (only worse in the sense that at least the current approach lets you copy the `?path=etc` params), but it'd be nice to use something that supports actual link functionality.

Reply:
Other than that, I like this better than our current approach 😊